src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -2367,11 +2374,13 @@ namespace WaterToAirHeatPumpSimple {
         QSensible *= PartLoadRatio;
         Winput *= RuntimeFrac;
         QSource = QSource_fullload * PartLoadRatio;
-
+        DataHeatBalance::HeatReclaimSimple_WAHPCoil(HPNum).AvailCapacity = QSource;
+        
         //  Add power to global variable so power can be summed by parent object
         DXElecCoolingPower = Winput;
 
         ReportingConstant = TimeStepSys * SecInHour;
+        QSource = QSource - DataHeatBalance::HeatReclaimSimple_WAHPCoil(HPNum).DesuperheaterReclaimedHeat;
```

Comment:
This is fine, but you are welcome to use the `-=` operator here if you want.

Reply:
Fixed